account_compensate_advance/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,5 @@
+* `Escodoo <https://www.escodoo.com.br>`_:
+
+  * Marcel Savegnago <marcel.savegnago@escodoo.com.br>
+  * Kaynnan Lemes <kaynnan.lemes@escodoo.com.br>
+  * Douglas Custodio <douglascstd@yahoo.com>
```

Comment:
aqui pode manter o da escodoo mesmo.. ele nao associa com a conta do git.. soh se for colocar como mantenedor do manifesto.. ai sim coloca o user do github.